src/main/webapp/src/components/handleError.js

```diff
@@ -1,19 +1,7 @@
-export default function(vm, error) {
-    const { $router, $root } = vm;
+export default function(vm /*, error */) {
```

Comment:
keine auskommentierten parameter

Reply:
das sollte den Platz für den Toast Fehlermeldung später in anderen Task.